config.js

```diff
@@ -411,11 +411,42 @@ config.DEFAULT_ACCOUNT_PREFERENCES = {
     ui_theme: 'DARK'
 };
 
-// load a local config file that overwrites some of the config
+//Load overrides if exist
 try {
+    // load a local config file that overwrites some of the config
     // eslint-disable-next-line global-require
     const local_config = require('./config-local');
     Object.assign(config, local_config);
+
+    //load env variables to override if exists (should start with CONFIG_JS_)
+    const ENV_PREFIX = 'CONFIG_JS_';
+    Object.keys(process.env).forEach(function(key) {
```

Comment:
I would use `for of` instead of `forEach`. a bit more readable and doesn't require a function:

```suggestion
    for (key of Object.keys(process.env)) {
```